spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Tracer.java

```diff
@@ -72,25 +82,33 @@
 	/**
 	 * Start a new span if the sampler allows it or if we are already tracing in this
 	 * thread. A sampler can be used to limit the number of traces created.
-	 *  @param name the name of the span
+	 *
+	 * @param name the name of the span
 	 * @param sampler a sampler to decide whether to create the span or not
 	 */
 	Span startTrace(String name, Sampler sampler);
 
 	/**
-	 * Pick up an existing span from another thread.
+	 * Pick up an existing span from another thread. Continuing a span signifies that
```

Comment:
Contributes to a span started in another thread. The returned span shares mutable state with the input.